src/transformers/models/t5/modeling_t5.py

```diff
@@ -640,6 +640,11 @@ def forward(
         hidden_states, present_key_value_state = self_attention_outputs[:2]
         attention_outputs = self_attention_outputs[2:]  # Keep self-attention outputs and relative position weights
 
+        # clamp inf values
+        if torch.isinf(hidden_states).any():
+            clamp_value = torch.finfo(hidden_states.dtype).max - 1000
```

Comment:
why the -1000?

Reply:
Just to be on the safer side, setting it to the exact max value might again lead to `inf` values in subsequent layers

Reply:
Okey just noticed that we do the same in Bart as well